docs/authentication/social-connections/tiktok.mdx

```diff
@@ -1,47 +1,75 @@
 
-To make the development flow as smooth as possible, Clerk uses  preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
+To make the setup process easier, it's recommended to keep two browser tabs open: one for your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [TikTok Developer Portal](https://developers.tiktok.com/).
 
-For production instances, you will need to create your own developer account with **TikTok** and generate your own Client ID and Client secret.
+<Steps>
+  ### Enable TikTok as a social connection in Clerk
 
-> [!NOTE]
-> The purpose of this guide is to help you set up a TikTok developer account  and a TikTok OAuth app - if you're looking for step-by-step instructions using Clerk to add social connection (OAuth) to your application, follow the [Social connection (OAuth) guide](/docs/authentication/social-connections/oauth).
+  1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+  1. Select **Add connection** and select **For all users**.
+  1. In the **Choose provider** dropdown, select **TikTok**.
+  1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on.
+  1. Save the **Authorized redirect URI** somewhere secure. Keep the modal and page open.
 
-## Before you start
+  ### Create your app in TikTok
 
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
-- You need to have a TikTok Developer account. To create one, visit the [TikTok for developers](https://developers.tiktok.com/) page and click on Login ➜ Sign Up
+  > [!TIP]
+  > If you're creating an app as an organization rather than as an individual developer, you must first [create an organization](https://developers.tiktok.com/organizations). For guidance on which option to choose, refer to the [Working with organizations guide](https://developers.tiktok.com/doc/working-with-organizations/).
 
-## Configuring TikTok social connection
+  1. On a separate page, go to the [TikTok Developer Portal](https://developers.tiktok.com/) and sign in.
+  1. In the top-right, select [**Developer Portal**](https://developers.tiktok.com/apps) then choose **Manage apps**. You'll be redirected to the **Manage apps** page.
+  1. Select **Connect an app**. Complete the form then select **Confirm**. You'll be redirected to your TikTok app's **Production** page.
+  1. In the **Basic information** section, complete the form.
+  1. Under **Platforms**, select your platform then paste the **Authorized Redirect URI** you saved from the Clerk Dashboard.
```

Comment:
It says link to the official website, it doesn't mention the redirect URI

<img width="987" alt="Screenshot 2024-10-15 at 10 26 12" src="https://github.com/user-attachments/assets/88594ddb-4322-44ab-b763-9beb602c4797">